content/en/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -1202,12 +1202,48 @@ Service:
 kubectl delete svc nginx
 ```
 
+Delete the persistent storage media for the PersistentVolumes used in this tutorial.
 
-{{< note >}}
-You also need to delete the persistent storage media for the PersistentVolumes
-used in this tutorial.
+```shell
+kubectl get pvc
+```
+```
+NAME        STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+www-web-0   Bound    pvc-2bf00408-d366-4a12-bad0-1869c65d0bee   1Gi        RWO            standard       25m
+www-web-1   Bound    pvc-ba3bfe9c-413e-4b95-a2c0-3ea8a54dbab4   1Gi        RWO            standard       24m
+www-web-2   Bound    pvc-cba6cfa6-3a47-486b-a138-db5930207eaf   1Gi        RWO            standard       15m
+www-web-3   Bound    pvc-0c04d7f0-787a-4977-8da3-d9d3a6d8d752   1Gi        RWO            standard       15m
+www-web-4   Bound    pvc-b2c73489-e70b-4a4e-9ec1-9eab439aa43e   1Gi        RWO            standard       14m
+```
 
+```shell
+kubectl get pv
+```
+```
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM               STORAGECLASS   REASON   AGE
+pvc-0c04d7f0-787a-4977-8da3-d9d3a6d8d752   1Gi        RWO            Delete           Bound    default/www-web-3   standard                15m
+pvc-2bf00408-d366-4a12-bad0-1869c65d0bee   1Gi        RWO            Delete           Bound    default/www-web-0   standard                25m
+pvc-b2c73489-e70b-4a4e-9ec1-9eab439aa43e   1Gi        RWO            Delete           Bound    default/www-web-4   standard                14m
+pvc-ba3bfe9c-413e-4b95-a2c0-3ea8a54dbab4   1Gi        RWO            Delete           Bound    default/www-web-1   standard                24m
+pvc-cba6cfa6-3a47-486b-a138-db5930207eaf   1Gi        RWO            Delete           Bound    default/www-web-2   standard                15m
+```
 
-Follow the necessary steps, based on your environment, storage configuration,
-and provisioning method, to ensure that all storage is reclaimed.
-{{< /note >}}
+```shell
+kubectl delete pvc www-web-0 www-web-1 www-web-2 www-web-3 www-web-4
+```
+
+```
+persistentvolumeclaim "www-web-0" deleted
+persistentvolumeclaim "www-web-1" deleted
+persistentvolumeclaim "www-web-2" deleted
+persistentvolumeclaim "www-web-3" deleted
+persistentvolumeclaim "www-web-4" deleted
+```
+
+```shell
+kubectl get pvc
```

Comment:
What about the PVs? Remember that the page says:

> If your cluster is not configured to do so, you will have to manually provision two 1 GiB volumes prior to starting this
tutorial.

at the start

Reply:
Hi @sftim Actually PV's will be successfully delete when i delete PVC's so should i explicitly mention output of `PV` status after cleaning up of pvc's i.e 

```
kubectl get pv 
No resources found
 
```

Reply:
Hi @sftim Could you please suggest me on  this.

> Hi @sftim Actually PV's will be successfully delete when i delete PVC's so should i explicitly mention output of `PV` status after cleaning up of pvc's i.e
> 
> ```
> kubectl get pv 
> No resources found
>  
> ```

Reply:
Hi @sftim could you please suggest on should i document  delete PV's step also?

Reply:
I think just a reminder at the end that if you manually provisioned volumes prior to the tutorial, it needs to be cleaned up manually should be sufficient

Reply:
@annajung Sure i will add the reminder at the end . Thanks for suggestions.

Reply:
@annajung @sftim  i have kept the note regarding the same. PTAL

Reply:
> Hi @sftim Actually PV's will be successfully delete when i delete PVC's so should i explicitly mention output of `PV` status after cleaning up of pvc's i.e
> 
> ```
> kubectl get pv 
> No resources found
>  
> ```

Others have replied about this, PVs will only be automatically deleted if the PVs were dynamically provisioned
PVs that were manually created will have to be deleted manually e.g. `kubectl delete pv pv1`